draft-ietf-privacypass-architecture.md

```diff
@@ -448,6 +448,51 @@ Metadata may also be arbitrarily long or bounded in length. The amount of
 permitted metadata may be determined by application or by the underlying
 cryptographic protocol.
 
+## Adding public metadata to the protocol
```

Comment:
I'd recommend removing this entire section and folding it into the second paragraph above. Specifically, I might note that a timestamp-based epoch or the client's ASN could be used as transparent public metadata.

Reply:
Changed ;)